apps/omg_child_chain/config/dev.exs

```diff
@@ -3,5 +3,4 @@ use Mix.Config
 
 config :omg_child_chain,
   ethereum_events_check_interval_ms: 500,
-  block_queue_eth_height_check_interval_ms: 1_000,
-  coordinator_eth_height_check_interval_ms: 1_000
```

Comment:
it was not used at all

Reply:
:+1: whoops! missed that or maybe a git-merge fail? thx for catching!